package.json

```diff
@@ -79,20 +79,26 @@
     "np": "^7.2.0",
     "prettier": "^2.2.1",
     "pretty-quick": "^3.1.3",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
     "require-from-string": "^2.0.2",
     "rollup": "^3.2.5",
     "rollup-plugin-clear": "^2.0.7",
     "rollup-plugin-visualizer": "~5.9.0",
     "storybook-chromatic": "^4.0.2",
     "ts-jest": "^26.5.0",
     "tslib": "^2.4.1",
-    "typescript": "^4.8.4",
+    "typescript": "^4.9.5",
     "vite": "3.2.4",
     "web-vitals": "^3.0.4"
   },
   "known_issues": {
+    "react@18.2.0": {
+      "@types/react": ["~18.0.38", "18.2.0"]
+    },
+    "react-dom@18.2.0": {
+      "@types/react-dom": ["~18.0.11", "18.2.0"]
+    },
```

Comment:
This syntax is undocumented, so worth improving errors from the package linter, @mxdvl